modules/serverless-ossm-v2x-jwt.adoc

```diff
@@ -1,13 +1,9 @@
-// Module included in the following assemblies:
-//
-// * serverless/networking/serverless-ossm-jwt.adoc
-
 [id="serverless-ossm-v2x-jwt_{context}"]
 = Using JSON Web Token authentication with {ProductShortName} 2.x and {ServerlessProductName}
 
 .Procedure
 
-. Create the `RequestAuthentication` resource in each serverless application namespace that is a member in the `ServiceMeshMemberRoll` object,  by copying the following code into a YAML file:
+. Create a `RequestAuthentication` resource in each serverless application namespace that is a member in the `ServiceMeshMemberRoll` object:
```

Comment:
Is _namespace that is a member in the `ServiceMeshMemberRoll` object_ the same as _namespace that is part of the Service Mesh member roll._? I like the latter wording better. Seems simpler.